components/Componenten/input-stepper/input-stepper.njk

```diff
@@ -0,0 +1,16 @@
+{% set localId = generateLocalId(prefix, id) %}
+
+<div class="dso-input-stepper">
+  {% if label %}
+    <label for="{{ localId }}" class="control-label">
+      {{ label }}
+    </label>
+  {% endif %} 
+  {% render '@button', {type: 'button', modifier: 'link', label: minusButtonLabel, icon: 'minus-square', iconOnly: 'true', disabled: minusButtonInactive} %}
+  <input type="number" 
```

Comment:
Als je dan toch kiest voor een `type="number"`, zou je de min en de max ook met native attributen nog kunnen markeren (jawel: `min` en `max`)...

Reply:
Dat kan, maar dat heeft alleen effect op standaardknopjes bij een `<input type="number">`. Maar die zijn hier weggestijld omdat we eigen buttons gebruiken. Als je bij de input -zoals hieronder voorgesteld- ook gebruikersinvoer toestaat, kan je nog steeds een waarde buiten de min-max range invoeren: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_max_min.

Dus of we maken er een standaard input-veld van, met duidelijke invoer context (bordertje) of we laten het zo als het is?